src/main/java/macrobase/analysis/pipeline/operator/OneToOneOperator.java

```diff
@@ -0,0 +1,3 @@
+package macrobase.analysis.pipeline.operator;
+
+public interface OneToOneOperator<S, T> extends MBConsumer<S>, MBProducer<T> { }
```

Comment:
What do you think about renaming this to just Operator? When we have multi operators we can specify that in the name